src/crypto.rs

```diff
@@ -7,12 +7,18 @@ use std::ops::{Deref, DerefMut};
 
 const KEY_GENERATOR: MacaroonKey = MacaroonKey(*b"macaroons-key-generator\0\0\0\0\0\0\0\0\0");
 
-// A convenience type for a MacaroonKey with helpful methods attached for
-// conversion. Using the default trait will return a randomly generated key
+/// A convenience type for a secret MacaroonKey with helpful methods attached for conversion. Using
```

Comment:
nit: I think "MacaroonKey" here can just be plain "macaroon key", otherwise it would be a little weird referring to the type itself in this context.

Reply:
Please ignore. I see the last commit updated this :)